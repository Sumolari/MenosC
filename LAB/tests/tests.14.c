{
	int a;
	a = 199;
	print(a);
}