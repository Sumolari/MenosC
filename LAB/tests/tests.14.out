  0 EASIG     i: 199  ,          ,  p:   1  
  1 EASIG     p:   1  ,          ,  p:   0  
  2 EASIG     p:   0  ,          ,  p:   2  
  3 EWRITE            ,          ,  p:   2  
  4 FIN               ,          ,          
